mcci-catena-provision-helium.py

```diff
@@ -1,899 +1,939 @@
+#!/usr/bin/env python3
+
+##############################################################################
+# 
+# Module: mcci-catena-provision-helium.py
+#
+# Function:
+#     Provision a catena device through Helium cli
+#
+# Copyright and License:
+#     This file copyright (c) 2021 by
+#
+#         MCCI Corporation
+#         3520 Krums Corners Road
+#         Ithaca, NY  14850
+#
+#     See accompanying LICENSE file for copyright and license information.
+#
+# Author:
+#     Sivaprakash Veluthambi, MCCI   May, 2021
```

Comment:
MCCI style doesn't use "," in "Month Year" sequences, only in "Month Day Year" sequences. Minor point, but might as well fix it now.

Reply:
@terrillmoore I have fixed the date format in header as per the MCCI style. Please check and merge the pull request.